jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt

```diff
@@ -89,7 +88,6 @@ class CloudFormationStackNode(project: Project, val stackName: String, private v
 
     override fun onDoubleClick(model: DefaultTreeModel, selectedElement: DefaultMutableTreeNode) {
         super.onDoubleClick(model, selectedElement)
-        openStack(nodeProject, stackName)
     }
```

Comment:
I think we can remove this method as it does nothing more.

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/956/files#diff-510cc4017b37da840e516696085651fbL23 - it's gone!